packages/@icon-magic/generate/src/index.ts

```diff
@@ -61,6 +61,6 @@ export async function generateFromConfigHash(
   iconConfig: IconConfigHash,
   hashing = true
 ): Promise<void> {
-  const iconSet = new IconSet(iconConfig, true);
+  const iconSet = new IconSet(iconConfig, hashing);
```

Comment:
oh nice, probably missed before.

Reply:
Yeah, think its one of those changes that have slipped in by mistake.